spikes/xml/claim.xml

```diff
@@ -0,0 +1,237 @@
+            <!--Car travel-->
+            <!--Parking-->
+            <!--Hotel accommodation-->
+            <!--Train/public transport-->
+            <!--Travel time-->
+            <!--Road or tunnel tolls-->
+            <!--Cab fares-->
+            <!--Subsistence-->
+            <date>2016-05-22</date>
+            <expense_type> Car travel</expense_type>
+            <location>Peterborough</location>
+            <quantity></quantity>
+            <rate></rate>
+            <amount>35.44</amount>
+            <distance>170</distance>
+            <mileage_rate>25p</mileage_rate>
+            <reason_type></reason_type>
+        </expense>
+    </expenses>
+
+
+    <disbursements>
+        <!--Zero or more disbursments-->
+        <disbursement>
+            <!--Disbursement types can be one of the following, or a code to represent one of the following:-->
+            <!--Accident reconstruction report-->
+            <!--Accounts-->
+            <!--Computer experts-->
+            <!--Consultant medical reports-->
+            <!--Costs judge application fee-->
+            <!--Costs judge preparation award-->
+            <!--DNA testing-->
+            <!--Engineer-->
+            <!--Enquiry agents-->
+            <!--Facial mapping expert-->
+            <!--Financial expert-->
+            <!--Fingerprint expert-->
+            <!--Fire assessor/explosives expert-->
+            <!--Forensic scientists-->
+            <!--Handwriting expert-->
+            <!--Interpreter-->
+            <!--Lip readers-->
+            <!--Medical expert-->
+            <!--Memorandum of conviction fee-->
+            <!--Meteorologist-->
+            <!--Other-->
+            <!--Overnight expenses-->
+            <!--Pathologist-->
+            <!--Photocopying-->
+            <!--Psychiatric reports-->
+            <!--Psychological report-->
+            <!--Surveyor/architect-->
+            <!--Transcripts-->
+            <!--Translator-->
+            <!--Vet report-->
+            <!--Voice recognition-->
+            <disbursement_types>Accident reconstruction report</disbursement_types>
+            <net_amount>9.70</net_amount>
+            <vat_amount>1.94</vat_amount>
+        </disbursement>
+    </disbursements>
+
+    <documents>
+        <!--Zero or more documents - always pdf-->
+        <document>
+            <uuid>345fa455-2526-47d9-9ca0-bcc4afddfa68</uuid>
+            <file_name>Hogdson_-_re_order.pdf</file_name>
+            <!--URI refers to location where document is stored in Amazon S3 bucket-->
+            <uri>documents/000/044/782/Hogdson_-_rep_order.pdf</uri>
+            <size>388031</size>
+        </document>
+    </documents>
+
+
+
+    <messages>
+        <!--Zero or more messages-->
+        <message>
+            <!--UUID of message sender - either caseworker or external user-->
+            <sender_uuid>345fa455-2526-47d9-9ca0-bcc4afddfa68</sender_uuid>
```

Comment:
I don't know if this UUD will be useful on its own, we might need to supply further details.